### ISSUE TITLE
UI makes examples harder to understand. 

### ISSUE THREAD

**Issue by chaseaucoin** [dotnet/mbmlbook#2]
Love the book, and the repo, but the examples have too much UI code especially for the basic examples. It takes focus away from actually learning how the varibles, properties, and engine work together because you have to slough through UI code and mentally parse whats there for machine learning and what is there for UI purposes.

**Comment by johnmwinn** [comment]
Thanks for the feedback.  Are there particular chapters that you think suffer from this problem or is it throughout?

**Comment by ricardoV94** [comment]
I disagree. So far (chapter3) all examples and illustrations felt rather informative and intuitive.

**Comment by mzhukovs** [comment]
I agree with OP. I can only imagine that Ricardo is referring to the material in the eBook/pdf itself, which includes zero reference to the corresponding sample code (at least through the first 2 chapters, which is where I'm at). The material is fantastic, but navigating and tying it in with the code is quite difficult and made more complicated thanks to the UI elements.
